Feature/sdt 399

helttonrl opened the pull request:
Se agregaron y se expusieron los métodos de timbrado v1 v2 v3 v4 y sus base64 para ser utilizados en visual basic 6, foxpro y delphi.

helttonrl committed:
SDT-399 Se añadieron las nuevas versiones de timbrado cpp

helttonrl committed:
SDT-399 Se expusieron metodos para v1 v2 v3 v4 en xml y en b64 incluyendo los que usa VB6

helttonrl committed:
SDT-399 Se cambió la versión para que generara un nuevo nuget.

helttonrl committed:
SDT-399 Se agregó otro proyecto de ejemplo de VB6.

helttonrl committed:
SDT-399 Se actualizó la imagen.

helttonrl committed:
SDT-399 Se mejoró la el ejemplo de visual basic 6.

alxcodes commented on sw-sdk-cpp-UT/StampTest.cpp:
```diff
@@ -14,54 +17,113 @@ namespace swsdkcppUT
 	{
 	private:
-		char *_url = "http://services.test.sw.com.mx";
+		char *_url = "http://swservicestest-rc.azurewebsites.net";
```
Esta url debe quitarse cuando se vaya a develop es para nosotros y el usuario final no la deberia de ver

alxcodes commented on sw-sdk-cpp.nuspec:
```diff
@@ -13,7 +13,7 @@
     <releaseNotes>Autenticación y timbrado</releaseNotes>
     <copyright>Copyright 2017</copyright>
-    <tags>CFDI  CFDI33 native stamp timbres rest</tags>
+    <tags>CFDI CFDI33 native stamp timbres rest b64 cpp vb6</tags>
```
si añades el nombre de vb6 por que no añadiste los de delphi y foxpro?

alxcodes commented on sw-sdk-cpp-UT/StampTest.cpp:
```diff
@@ -14,54 +17,113 @@ namespace swsdkcppUT
 		}
-
-		TEST_METHOD(Stamp_V2)
+		TEST_METHOD(UT_Stamp_V2)
```
Los nombres de los unitest deberian estar asociados al metodo que se esta probando en este caso se esta probando StampByTokenV2 pero el nombre del test solo dice StampV2

alxcodes committed:
-Add Stamp Base64 to Delphi Example
-Add Somes UnitTest

helttonrl committed:
SDT-399 Se modificó el proyecto de vb.

helttonrl committed:
SDT-399 Se resolvieron confluctos.

Commit:
SDT-399 se agrego proyecto de UT para visual fox pro

alxcodes committed:
Create Response 401 & 404 for c++
Add Unitest
Create Base64 from string for Unitest
Modify Boundary and now Use GUID item for that
Some Adjust Design for Sample Delphi

helttonrl committed:
SDT-399 Se agregaron pruebas unitarias para un xml con 1028 conceptos eb b64 tambien se cambiaron las pruebas unitarias para que aunque el xml esté mal de todas maneras evaluara el resultado como correcto.

helttonrl committed:
SDT-399 Se cambió la URL del timbrado.

alxcodes committed:
Change Repo Samples
